File: tests/data-clumps/test-cases/field-field/always-class/positive/typescript/source/Patient.ts
import { Address } from './Address';
import { ContactInfo } from './ContactInfo';
import { Insurance } from './Insurance';

export class Patient {
  public address: Address;
  public contactInfo: ContactInfo;
  public insurance: Insurance;
  public patientId: string;

  constructor(
    address: Address,
    contactInfo: ContactInfo,
    insurance: Insurance,
    patientId: string
  ) {
    this.address = address;
    this.contactInfo = contactInfo;
    this.insurance = insurance;
    this.patientId = patientId;
  }
}

File: tests/data-clumps/test-cases/field-field/always-class/positive/java/source/Patient.java
public class Patient {
    public Address address;
    public ContactInfo contactInfo;
    public Insurance insurance;
    public String patientId;

    public Patient(Address address, ContactInfo contactInfo, Insurance insurance, String patientId) {
        this.address = address;
        this.contactInfo = contactInfo;
        this.insurance = insurance;
        this.patientId = patientId;
    }
}
